packages/paste-website/src/pages/components/card/index.mdx

```diff
@@ -138,15 +141,36 @@ and primary action ([Button](/components/button)) together. Relating these three
 digest and provides a clear call to action. Padding surrounding the inner content of a Card can be
 adjusted to suit the needs of your implementation.
 
-<LivePreview scope={{Card, Paragraph, Heading, Button}} language="jsx">
-  {`<Card padding="space120">
-  <Heading as="h2" variant="heading20">This is a card with a primary action</Heading>
-  <Paragraph>
-    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam dui tellus, tristique in tincidunt cursus, efficitur
-    at felis. Phasellus imperdiet, lorem et commodo egestas, arcu.
-  </Paragraph>
-  <Button variant="submit"> I am a primary action button </Button>
-</Card>`}
+<LivePreview scope={{Stack, Card, Paragraph, Heading, Button}} language="jsx">
+  {`<Stack orientation="vertical" spacing="space40">
+    <Card padding="space120">
+      <Heading as="h2" variant="heading20">The Transgender District</Heading>
+      <Paragraph>
+        The mission of the Transgender District is to create an urban environment that fosters the rich history, culture, legacy, and empowerment of transgender
+        people and its deep roots in the southeastern Tenderloin neighborhood. The transgender district aims to stabilize and economically empower the transgender
+        community through ownership of homes, businesses, historic and cultural sites, and safe community spaces.
+      </Paragraph>
+      <Button as="a" href="https://www.transgenderdistrictsf.com/">Support The Transgender District</Button>
```

Comment:
All these example Buttons should be Anchors otherwise the code will fail. One of the problems with the LivePreview component is that it won't show you our usual warnings. We're working on something to fix that.

Reply:
Ahh I was going for a button-style anchor here (like so: https://paste.twilio.design/components/button/#buttons-with-link-functionality). Is this markup wrong?

Reply:
In that case it's still missing the variant prop

Reply:
Ahh, yes it is! I'll get that added. Thanks!